### PR TITLE
Adding direct link in Support section

### PR DESCRIPTION
**Pull request by malivinayak**
Finding a respective package for particular platform is bit tricky in Releases section. Either mention a links in Support Section or in README.md such that easy to find.
Adding a android-x86_64-release.apk link for android.

<hr>

BTW I tried this one too what's your opinion
## Support 

<details>
  <summary>✅android</summary>
  
  1. [android-arm32.apk](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-arm32-release.apk)
  2. [android-arm64.apk](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-arm64-release.apk)
  3. [android-x86_64.apk](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-x86_64-release.apk)
  </details>
  
🔳 ios
✅ [web](https://whats-that-anime.outdatedguy.rocks/)
🔳 windows
🔳 macos
🔳 linux

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,9 +37,9 @@ An **Image Search** App to find **Anime** Details related to the Image. Created
 
 This package currently is only tested on android and the web platform.
 
-- [x] android
+- [x] [android](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-x86_64-release.apk)
```
Comment (OutdatedGuy) [inline README.md:40]: Linking to only one architecture is a bad idea as all phones uses different chipsets.

**Review by OutdatedGuy — CHANGES_REQUESTED** [review]
The **dropdown** idea in the pull request seems very nice but can you make it more attractive?

**Comment by malivinayak** [comment]
ohk I will try it.

**Comment by malivinayak** [comment]
Reopening PR with requested changes
I tried by best to make it attractive take a look

**Comment by malivinayak** [comment]
## Support

This package currently is only tested on android and the web platform.
<details>
  <summary>✅ android</summary>
  
  1. [android-arm32.apk](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-arm32-release.apk)
  2. [android-arm64.apk](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-arm64-release.apk)
  3. [android-x86_64.apk](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-x86_64-release.apk)
</details>

<details>
  <summary><a href="https://whats-that-anime.outdatedguy.rocks/">✅ web</a></summary>
  
  * [whats-that-anime.outdatedguy.rocks](https://whats-that-anime.outdatedguy.rocks/)
</details> 

🔳 ios 
🔳 windows 
🔳 macos 
🔳 linux 

<hr> <hr> <br>
In this for android you can choose from drop down and for web you can directly click on web as it is embedded with link or you can expand the drop down and then click on  whats-that-anime.outdatedguy.rocks
<br> <br>

File: README.md [hunk 2]
```diff
@@ -36,13 +36,25 @@ An **Image Search** App to find **Anime** Details related to the Image. Created
 ## Support
 
 This package currently is only tested on android and the web platform.
+<details>
+  <summary>✅ android</summary>
+  
+  1. [android-arm32.apk](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-arm32-release.apk)
```
Comment (OutdatedGuy) [inline README.md:42]: Instead of giving fixed version number, use the auto latest release link.

File: README.md [hunk 3]
```diff
@@ -36,13 +36,25 @@ An **Image Search** App to find **Anime** Details related to the Image. Created
 ## Support
 
 This package currently is only tested on android and the web platform.
+<details>
+  <summary>✅ android</summary>
```
Comment (OutdatedGuy) [inline README.md:40]: Use bullets instead of numbering here using -

File: README.md [hunk 4]
```diff
@@ -36,13 +36,25 @@ An **Image Search** App to find **Anime** Details related to the Image. Created
...
+  
+  1. [android-arm32.apk](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-arm32-release.apk)
+  2. [android-arm64.apk](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-arm64-release.apk)
+  3. [android-x86_64.apk](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-x86_64-release.apk)
+</details>
+
+<details>
+  <summary><a href="https://whats-that-anime.outdatedguy.rocks/">✅ web</a></summary>
```
Comment (OutdatedGuy) [inline README.md:48]: Put _tick mark_ and `space` between _web_ outside the `anchor` tag

File: README.md [hunk 5]
```diff
@@ -36,13 +36,25 @@ An **Image Search** App to find **Anime** Details related to the Image. Created
 ## Support
```
Comment (OutdatedGuy) [inline README.md:36]: Rename this as **Platform Support**

File: README.md [hunk 6]
```diff
@@ -36,13 +36,25 @@ An **Image Search** App to find **Anime** Details related to the Image. Created
...
+  
+  1. [android-arm32.apk](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-arm32-release.apk)
+  2. [android-arm64.apk](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-arm64-release.apk)
+  3. [android-x86_64.apk](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-x86_64-release.apk)
+</details>
+
+<details>
+  <summary><a href="https://whats-that-anime.outdatedguy.rocks/">✅ web</a></summary>
+  
+  * [whats-that-anime.outdatedguy.rocks](https://whats-that-anime.outdatedguy.rocks/)
```
Comment (OutdatedGuy) [inline README.md:50]: Use `-` for bullet points

**Review by OutdatedGuy — CHANGES_REQUESTED** [review]
I have added few small changes notes. Please do those and I'll merge it.

File: README.md (hunk 6 above)
Comment (malivinayak) [reply README.md:50]: Done

File: README.md [hunk 7]
```diff
@@ -33,16 +33,28 @@ An **Image Search** App to find **Anime** Details related to the Image. Created
...
 - Customize App Behaviour from Settings Page
 
-## Support
+## Platform Support
 
 This package currently is only tested on android and the web platform.
+<details>
+  <summary>✅ android</summary>
+  
+  + [android-arm32.apk](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-arm32-release.apk)
```
Comment (OutdatedGuy) [inline README.md:42]: Why does it use `+` instead of `-` here?

File: README.md [hunk 8]
```diff
@@ -33,16 +33,28 @@ An **Image Search** App to find **Anime** Details related to the Image. Created
...
+  + [android-arm64.apk](https://github.com/OutdatedGuy/Whats-That-Anime/releases/download/v2.0.0/android-arm64-release.apk)
```
Comment (OutdatedGuy) [inline README.md]: Link to latest release goes like this
```
https://github.com/<account name>/<repo name>/releases/latest/download/<file name>
```

File: README.md (hunk 7 above)
Comment (malivinayak) [reply README.md]: Check Now

File: README.md (hunk 8 above)
Comment (malivinayak) [reply README.md]: Thank You for this detail
I am unaware about this

**Comment by malivinayak** [comment]
I have made all request changes 
Take a look

**Comment by malivinayak** [comment]
☺️ I learned lot while making this PR.
Thank You for considering ✅

**Comment by malivinayak** [comment]
Thank You 😊

**Pull request merged**